Tests/SentryTests/SentryHubTests.swift

```diff
@@ -394,6 +394,19 @@ class SentryHubTests: XCTestCase {
         XCTAssertEqual(1, fixture.client.captureSessionInvocations.count)
     }
 
+    func testCaptureErrorBeforeSession() {
```

Comment:
`h`: We must set the `_errorsBeforeSession` to `0` in `SentryHub.endSession`. Users could disable auto session tracking and manually track sessions. Please also add a test for this.